test/api/test_c_op_construct.c

```diff
@@ -30,7 +30,7 @@ void expect_equal(const char* x, const char* y)
         abort();
 }
 
-int main()
+int main(void)
```

Comment:
Void necessary ?

Reply:
Yes, that is C file (not C++). C standard does not have prototype `int main()`. The compilation is failing here.

Reply:
Yea this is fixing a clang warning. I am not sure why we dont catch it in our CI though.